File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupX6.scala
package foo
import utest._
object GroupX6 extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Perseus", 34) }
    test("test2") { testGreeting("Quirinus", 32) }
    test("test3") { testGreeting("Rhea", 33) }
  }
} 

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupX6.kt
package foo

import org.junit.jupiter.api.Test

class GroupX6 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Perseus", 34)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Quirinus", 32)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Rhea", 33)
    }
} 
